### PR TITLE
Update help window and fix bugs in UG

### PR DESCRIPTION
**Pull request by mukundrs**
- Update help window with latest version of the application
- Fix inconsistencies in UG

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -426,32 +426,32 @@ Action | Format, Examples
...
+**Edit Reservation** | *Format* <br> `edit -rs/ -id/<MEMBER_ID + RESERVATION_ID> [-dt/<DATE_TIME>][-rm/<REMARK>]` <br> *Example* <br> `edit -rs/ -id/10001100001 -dt/2021-12-01 13:00 -rm/3 people`
 **Sort Members by Credit in Ascending Order** | *Format* <br> `sort -mem/ -c/ -a/` <br> *Example* <br> `sort -mem/ -c/ -a/`
 **Sort Members by Credit in Descending Order** | *Format* <br> `sort -mem/ -c/ -d/` <br> *Example* <br> `sort -mem/ -c/ -d/`
-**Redeem Point from Member by Member Id** | *Format* <br> `redeem -rd/ <POINTS> -id/ <MEMBER_ID>` <br> *Example* <br> `redeem -rd/ 100 -id/ 10006`
-**Redeem Point from Member by Member Index** | *Format* <br> `redeem -rd/ <point> -i/ <INDEX>` <br> *Example* <br> `redeem -rd/ 100 -i/ 1`
+**Redeem Point from Member by Member Id** | *Format* <br> `redeem -rd/<POINTS> -id/<MEMBER_ID>` <br> *Example* <br> `redeem -rd/100 -id/10006`
+**Redeem Point from Member by Member Index** | *Format* <br> `redeem -rd/<point> -i/<INDEX>` <br> *Example* <br> `redeem -rd/100 -i/1`
 **Summary** | *Format* <br> `summary` <br> **NOTE**: Only managers are allowed to view summary. Login as manager before entering command.
 **Clear Program** | `clear`
```
Comment (zzybluebell) [inline docs/UserGuide.md:457]: That is good

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 2]
```diff
@@ -17,18 +17,24 @@
...
+            + "Add transaction: add -txn -b/<BILLING> -id/<MEMBER_ID>\n"
+            + "Delete transaction: del -txn/ -id/<MEMBER_ID + TRANSACTION_ID>\n"
+            + "Edit transaction: edit -txn/ -id/<MEMBER_ID + TRANSACTION_ID> -b/<BILL_AMMOUNT>\n"
+            + "Add Reservation: add -rs/ -dt/<DATE_TIME yyyy-MM-dd HH:mm> -rm/ <REMARK> -id/<MEMBER_ID>\n"
+            + "Delete Reservation: del -rs/ -id/<MEMBER_ID + RESERVATION_ID>\n"
+            + "Edit reservation: edit -rs/ -id/<MEMBER_ID + RESERVATION_ID> [-dt/<DATE_TIME>][-rm/<REMARK>]\n"
+            + "Sort members by ascending credit: sort -mem/ -c/ -a/\n"
+            + "Sort members by descending credit: sort -mem/ -c/ -d/\n"
+            + "Redeem points: redeem -rd/<POINTS> -id/<MEMBER_ID>\n"
+            + "View summary: summary"
```
Comment (zzybluebell) [inline src/main/java/seedu/address/ui/HelpWindow.java:37]: Well down

File: docs/UserGuide.md [hunk 3]
```diff
@@ -75,15 +75,15 @@ Continue reading to explore the wonders of ezFoodie and enhance the way you mana
 **:information_source: Notes about the command format:**<br>
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `find -mem -n <NAME>`, `NAME` is a parameter which can be used as `find -mem -n John Doe`.
+  e.g. in `find -mem/-n/<NAME>`, `NAME` is a parameter which can be used as `find -mem -n/John Doe`.
```
Comment (zzybluebell) [inline docs/UserGuide.md:78]: what is /-n/?
should have a space between?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -75,15 +75,15 @@ Continue reading to explore the wonders of ezFoodie and enhance the way you mana
...
 * Items in square brackets are optional.<br>
-  e.g `list -mem [-pg <PAGE>]` can be used as `list -mem -pg 1` or as `list -mem`.
+  e.g `list -mem/[-pg <PAGE>]` can be used as `list -mem/-pg/1` or as `list -mem/`.
 
-* The `list` and `find` only show top 50 records by default. Pagination can be used `[-pg <PAGE>]` as optional value to choose different pages.
+* The `list` and `find` only show top 50 records by default. Pagination can be used `[-pg/<PAGE>]` as optional value to choose different pages.
 
 * If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  **e.g.** if you specify `-p 12341234 -p 56785678`, only `-p 56785678` will be taken.
+  **e.g.** if you specify `-p/12341234 -p/56785678`, only `-p/56785678` will be taken.
```
Comment (zzybluebell) [inline docs/UserGuide.md:86]: the space

File: docs/UserGuide.md [hunk 5]
```diff
@@ -75,15 +75,15 @@ Continue reading to explore the wonders of ezFoodie and enhance the way you mana
 **:information_source: Notes about the command format:**<br>
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `find -mem -n <NAME>`, `NAME` is a parameter which can be used as `find -mem -n John Doe`.
+  e.g. in `find -mem/-n/<NAME>`, `NAME` is a parameter which can be used as `find -mem -n/John Doe`.
 
 * Items in square brackets are optional.<br>
-  e.g `list -mem [-pg <PAGE>]` can be used as `list -mem -pg 1` or as `list -mem`.
+  e.g `list -mem/[-pg <PAGE>]` can be used as `list -mem/-pg/1` or as `list -mem/`.
```
Comment (zzybluebell) [inline docs/UserGuide.md:81]: the ""space"" between -mem/ and -pg/

**Review by stephanie-csy — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 6]
```diff
@@ -77,17 +77,17 @@ This User Guide aims to:
 
    * `list -mem/` : Lists all members.
 
-   * `add -mem/ -n/ John Doe -p/ 98765432 -e/ johndoe@gmail.com -a/ 112 Amoy Street, 069907, Singapore` : Adds a contact named `John Doe` to the member list.
+   * `add -mem/ -n/John Doe -p/98765432 -e/johndoe@gmail.com -a/112 Amoy Street, 069907, Singapore` : Adds a contact named `John Doe` to the member list.
```
Comment (Morrow1ndy) [inline docs/UserGuide.md:80]: Nice change to follow the same format we agreed.

File: docs/UserGuide.md [hunk 7]
```diff
@@ -356,45 +351,45 @@ Deletes a transaction.
...
+Example: `edit -txn/ -id/10001100002 -b/ 10.00`
 
 **Note :** Transaction date and time cannot be modified once a transaction is added.
 
 ### 14. Adding a reservation: `add -rs/`
 
 Adds a reservation to a member.
 
-Format: `add -rs/ -dt/ <DATE_TIME yyyy-MM-dd HH:mm> -rm/ <REMARK> -id/ <MEMBER_ID>`
+Format: `add -rs/ -dt/<yyyy-MM-dd HH:mm> -rm/ <REMARK> -id/ <MEMBER_ID>`
```
Comment (Morrow1ndy) [inline docs/UserGuide.md:372]: Good to delete unnecessary String here.

File: docs/UserGuide.md [hunk 8]
```diff
@@ -476,32 +471,32 @@ Action | Format, Examples
...
+**Edit Reservation** | *Format* <br> `edit -rs/ -id/<MEMBER_ID + RESERVATION_ID> [-dt/<DATE_TIME>][-rm/<REMARK>]` <br> *Example* <br> `edit -rs/ -id/10001100001 -dt/2021-12-01 13:00 -rm/3 people`
 **Sort Members by Credit in Ascending Order** | *Format* <br> `sort -mem/ -c/ -a/` <br> *Example* <br> `sort -mem/ -c/ -a/`
 **Sort Members by Credit in Descending Order** | *Format* <br> `sort -mem/ -c/ -d/` <br> *Example* <br> `sort -mem/ -c/ -d/`
-**Redeem Point from Member by Member Id** | *Format* <br> `redeem -rd/ <POINTS> -id/ <MEMBER_ID>` <br> *Example* <br> `redeem -rd/ 100 -id/ 10006`
-**Redeem Point from Member by Member Index** | *Format* <br> `redeem -rd/ <point> -i/ <INDEX>` <br> *Example* <br> `redeem -rd/ 100 -i/ 1`
+**Redeem Point from Member by Member Id** | *Format* <br> `redeem -rd/<POINTS> -id/<MEMBER_ID>` <br> *Example* <br> `redeem -rd/100 -id/10006`
+**Redeem Point from Member by Member Index** | *Format* <br> `redeem -rd/<point> -i/<INDEX>` <br> *Example* <br> `redeem -rd/100 -i/1`
 **Summary** | *Format* <br> `summary` <br> **NOTE**: Only managers are allowed to view summary. Login as manager before entering command.
 **Clear ezFoodie Data** | `clear`
```
Comment (Morrow1ndy) [inline docs/UserGuide.md:502]: Good.

**Review by Morrow1ndy — COMMENTED** [review]
LGTM

**Pull request merged**